less/components/consonant/cards/horizontal.less

```diff
@@ -0,0 +1,51 @@
+.consonant-CardsGrid {
+    .consonant-Card[class ^= "horizontal"] {
+        display: flex;
+        width: 100%;
+        height: @consonant-HorizontalCard-height;
+        align-items: center;
+        flex-direction: row;
+        .font(@color: @consonantDarkGrey800);
+    }
+
+    .consonant-Card {
```

Comment:
These changes should be breaking and override defaults for all non horizontal cards within any grid.